src/promptflow-core/promptflow/core/_serving/constants.py

```diff
@@ -4,3 +4,5 @@
 
 FEEDBACK_TRACE_FIELD_NAME = "feedback"
 FEEDBACK_TRACE_SPAN_NAME = "promptflow-feedback"
+
+PF_BUILTIN_TRACE_EXPORTERS_DISABLE = "PF_BUILTIN_TRACE_EXPORTERS_DISABLE"
```

Comment:
Why don't we use the env PF_TRACING_SKIP_LOCAL_SETUP_ENVIRON ?

Reply:
Purpose is different and this naming "PF_TRACING_SKIP_LOCAL_SETUP_ENVIRON " is confusing for customer. Here what they need is to disable the deployed endpoint's trace exporters even if there are APPLICATIONINSIGHTS_CONNECTION_STRING or OTEL_EXPORTER_OTLP_ENDPOINT configured. A separate name will make things more clear and easy to understand.

Reply:
If you mean using PF_TRACING_SKIP_LOCAL_SETUP_ENVIRON to disable the setup, this "PF_TRACING_SKIP_LOCAL_SETUP_ENVIRON" doesn't work for setup_exporter_from_environ.

Reply:
I see, so the exporter would also be different here.